todo_app/app.py

```diff
@@ -1,51 +1,53 @@
+import os
 from flask import Flask, render_template, url_for, redirect
 from flask.globals import request
-from todo_app.data.session_items import get_item, get_items, add_item, save_item, remove_item
-from todo_app.trello import Trello
+from todo_app.data.session_items import get_task, get_tasks, add_task, update_task_status
+from todo_app.mongo_service import MongoService
 from todo_app.views import ViewModel
 
 
-def create_app():
+def create_app(db_name = os.environ.get("DB_NAME")):
```

Comment:
Given that you're expecting to pick this variable up from an environment variable anyway this feels a bit inconsistent - would it be simpler to let this be picked up in the MONGO_SERVICE like the other variables, and have your end-to-end test set the relevant variable before creating the app with `os.environ['DB_NAME'] = 'test-db'`?